src/components/battery/BatteryController.h

```diff
@@ -39,6 +42,7 @@ namespace Pinetime {
       static constexpr nrf_saadc_input_t batteryVoltageAdcInput = NRF_SAADC_INPUT_AIN7;
       uint16_t voltage = 0;
       uint8_t percentRemaining = 0;
+      uint8_t lastPercentRemaining = 0;
```

Comment:
I'd replace this with a `bool batteryLowNotified = false;` to make the variable and conditions more explicit

Reply:
Thank you! I will do this.

Reply:
Resetting the flag if the watch is loading is enough, right? :grin:

Reply:
I think so, yes :+1: